### ISSUE TITLE
Cannot read property 'getPermissionStatus' of undefined

### ISSUE THREAD

**Issue by Aqueelmiq** [zoontek/react-native-permissions#183]
getting this issue on React native permissions 1.0.1 and React Native 0.50

I followed the instructions, ran react native link and added permissions on Xcode

I get this on doing
```
        Permissions.check('photo')
            .then(response => {
                //response is one of: 'authorized', 'denied', 'restricted', or 'undetermined'
                alert(JSON.stringify(response))
            });
```

This is copied right from the Readme.md

Screenshot:
![image](https://user-images.githubusercontent.com/6711678/32986223-5d83b3f6-cc82-11e7-9305-8667c79c400b.png)

**Comment by AdrianBrown1** [comment]
same ! can we get any info on this ?

**Comment by zoontek** [comment]
@Aqueelmiq @AdrianBrown1 Did you check if the project was correctly linked?

**Comment by Aqueelmiq** [comment]
@zoontek I tried both auto linking and manual linking. I also verified that the xcode project had the associated libraries and files.

**Comment by Aqueelmiq** [comment]
I think this issue only happens in react-native 0.50

**Comment by zoontek** [comment]
@Aqueelmiq I just update the example with the latest version of react-native and I can't reproduce the error (check the /example folder). Weird.

**Comment by Aqueelmiq** [comment]
@zoontek I just cloned your example app and it seems to work, with the exact same code that I have so I am guessing it's either a conflict with one of my other dependencies or some setup issue, I will look into it. You can close the issue.

**Comment by zoontek** [comment]
@Aqueelmiq I might have solved your problem with RN 0.50. I'm waiting for a npm publish

**Comment by zoontek** [comment]
@Aqueelmiq It's available! https://www.npmjs.com/package/react-native-permissions
